lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -143,6 +190,7 @@ export default class LockMonitor {
     };
 
     currentLockState.currentValueTimeLock = await this.lockResolver.resolveLockIdentifierAndThrowOnError(lastLockIdentifier);
+    currentLockState.status = LockStatus.Confirmed;

```

Comment:
Maybe I misunderstood, the comment above about "which we still need to resolve." is confusing to me, I was under the impression that we don't need to resolve, but we are doing it out of convenience?

Reply:
Updating the comment.